base/ext/push_back.h

```diff
@@ -4,22 +4,17 @@
 
 namespace ext
 {
-
 /// Moves all arguments starting from the second to the end of the vector.
 /// For example, `push_back(vec, a1, a2, a3)` is a more compact way to write
 /// `vec.push_back(a1); vec.push_back(a2); vec.push_back(a3);`
 /// This function is like boost::range::push_back() but works for noncopyable types too.
-template <typename T>
-void push_back(std::vector<T> &)
+template <class T, class ...Args>
+void push_back(std::vector<T> & vec, Args&& ...args)
```

Comment:
This method is no longer be presented in master because we will remove ext, consider to replace to some local function.